### ISSUE TITLE
node_modules/@excalidraw/excalidraw/types library failing giving errors on compile in angular

### ISSUE THREAD

**Issue by Shivugethub** [excalidraw/excalidraw#4013]
using excalidraw within angular application

packages
 ```
   "@excalidraw/excalidraw": "^0.9.0",
    "react": "^16.14.0",
    "react-dom": "^16.14.0"
```

![image](https://user-images.githubusercontent.com/33868554/135133908-98d2d4bd-439c-4bbc-b4db-86fbc6adc9ed.png)

**Comment by dwelle** [comment]
Unclear what's going on here. I've gotten similar errors but in the context of our eslint trying to lint the type definition files — unclear why your linter would try to do the same considering it's in node_modules.

**Comment by Shivugethub** [comment]
yes i am getting these errors if use tsx file, if i use js file, on compile i getting these below errors,

![image](https://user-images.githubusercontent.com/33868554/135260195-00efc69f-b5dc-4f84-b917-c07371b43f45.png)

**Comment by dwelle** [comment]
That doesn't make sense to me at all since we don't publish source files.

When you say "use tsx/js file", what you mean exactly?

**Comment by Shivugethub** [comment]
I have shared my repo here [https://github.com/Shivugethub/angular-reactjs-sample](url) which contains code and here is the errors got on running application

![image](https://user-images.githubusercontent.com/33868554/135411163-60eefa53-1b0c-4068-bde9-979bc37362ef.png)

**Comment by Shivugethub** [comment]
in angular application react-js working fine without excalidraw but after adding excalidraw getting issues from node-modules. 
"jfyi" no as been used webpack could please look into the issue

**Comment by ad1992** [comment]
Can you share Codesandbox link ? Also since the App is using angular, instead of bundling react, react-dom to your main bundle you might want to use directly in browser as mentioned in [usage in browser](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#in-browser) and you don't have to worry about web bundler as well

**Comment by Shivugethub** [comment]
via browser it's worked, Thank you, but via bundling it's still failing, [Codesandbox via bundle](https://codesandbox.io/s/xtjyq?file=/README.md). Here i have provided link if someone stuck or need help they can refer, #angular project link : [Codesandbox via borwser](https://codesandbox.io/s/7hbz0).
